mayhem/fuzz_plumber.py

```diff
@@ -0,0 +1,50 @@
+#! /usr/bin/env python3
```

Comment:
Exception handling is non-deterministic.

Reply:
Made deterministic using an iteration counter that defers exception handling